README.md

```diff
@@ -1,5 +1,5 @@
 [![Build Status](https://travis-ci.org/andela/reactors-cards-for-humanity.svg?branch=travisci-badge)](https://travis-ci.org/andela/reactors-cards-for-humanity)
-
+[![HoundCI](https://img.shields.io/badge/HoundCI-Integrated-blue.svg)](https://houndci.com/)
```

Comment:
No need for this. Just remove it and mark the task as done. It doesn't serve any purpose. (Make a comment in the story.)